modules/mobile/docs/schemas/CommunityCareProviders.yml

```diff
@@ -0,0 +1,9 @@
+type: object
```

Comment:
docs look to be missing the attributes and type field.

Reply:
Thank you for pointing this out. It led me to find some other small issues as well.